code/__HELPERS/unsorted.dm

```diff
@@ -953,22 +959,37 @@ var/global/image/busy_indicator_hostile
 			else
 				air_master.tiles_to_update += T2*/
 
-proc/DuplicateObject(obj/original, var/perfectcopy = 0 , var/sameloc = 0)
-	if(!original)
-		return null
 
-	var/obj/O = null
+/proc/DuplicateObject(atom/original, atom/newloc)
+	if(!original || !newloc)
```

Comment:
This `newloc` check is leaving `copy_contents_to()` broken. Granted, the proc is only used for the holodeck code, but it shouldn't be left like that. Just include `newloc` in the calls to `DuplicateObject` in `/area/proc/copy_contents_to(var/area/A , var/platingRequired = 0 )`.

Reply:
B-But you told me it's not used anywhere! Reeee